docs/Examples/Image/Main.purs

```diff
@@ -0,0 +1,36 @@
+module Examples.Image.Main where
+
+import Prelude
+
+import Effect.Console as Console
+import Node.FS.Aff as FS
+import HTTPure as HTTPure
+
+-- | Serve the example server on this port
+port :: Int
+port = 8090
+
+-- | Shortcut for `show port`
+portS :: String
+portS = show port
+
+-- | The path to the file containing the response to send
+filePath :: String
+filePath = "./docs/Examples/Image/circle.png"
+
+-- | Respond with image data when run
+image :: HTTPure.Request -> HTTPure.ResponseM
+image _ =
+  FS.readFile filePath >>= HTTPure.binaryResponse' 200 headers
+  where
+    headers = HTTPure.header "Content-Type" "image/png"
```

Comment:
We should probably add an analogue to Express's `https://expressjs.com/en/api.html#res.sendFile` to automatically set this header (not necessary for this PR obviously, I'll open a separate ticket)